Mantle/NSDictionary+MTLJSONKeyPath.h

```diff
@@ -22,6 +23,6 @@
 ///
 /// Returns the value for the key path which may be nil. Clients should inspect
 /// the success parameter to decide how to proceed with the result.
-- (id)mtl_valueForJSONKeyPath:(NSString *)JSONKeyPath success:(BOOL *)success error:(NSError **)error;
+- (id)mtl_valueForJSONKeyPath:(id)JSONKeyPath success:(BOOL *)success error:(NSError **)error;
```

Comment:
Can we use a protocol here that is only implemented by `NSString` and `MTLJSONKeyPath`?